_build.ts

```diff
@@ -101,22 +101,40 @@ console.log(
   `${colors.bold(colors.green("Generating"))} lib JS bindings...`,
 );
 
+const loader = `let wasmInstantiatePromise;
+switch (wasm_url.protocol) {
+  case "file:": {
+    const wasmCode = await Deno.readFile(wasm_url);
+    wasmInstantiatePromise = WebAssembly.instantiate(wasmCode, imports);
+    break;
+  }
+  case "https:":
+  case "http:": {
+    if ("permissions" in Deno) Deno.permissions.request({ name: "net", host: wasm_url.host });
```

Comment:
Should we do the same thing with the `readFile` above? I noticed it was removed below for some reason? (maybe accidentally?)

Reply:
It wasn't accidental.  I realised that the only thing I could do was ask for read permissions concisely and asking for permission to everything felt "wrong", but now that you bring it up again, I realise I can pass the `wasm_url` as the `path` and only request read permission for that.